### ISSUE TITLE
How to add and finish a order in Client folder

### ISSUE THREAD

**Issue by mithang** [stepanbenes/api-for-nopcommerce#93]
Help me

**Comment by mithang** [comment]
Please

**Comment by stepanbenes** [comment]
I don't have any idea what do you want to do. Add some detailed description.

**Comment by mithang** [comment]
@stepanbenes 
In the 'ClientApp' folder, I can't find a way to create/update/delete an order from the API, which includes shipping, payment, etc. Can you guide me on this? Thank you very much.

**Comment by stepanbenes** [comment]
There is an example of creating and updating a product. Just open the project in e.g. Visual Studio, use the intellisense to explore the methods on nopApiClient. There are `CreateOrder`, `UpdateOrder` and `DeleteOrder` methods.

**Comment by D3bo** [comment]
I suggest you to install the plugin and navigate to the page /api/swagger. I admit that I have never used the clientApp project and the test projects. 

Probably we should just removed them and if I have time create a new test project using more recent frameworks (It still uses .Net framework 4.6!). @stepanbenes

**Comment by stepanbenes** [comment]
> Probably we should just removed them and if I have time create a new test project using more recent frameworks (It still uses .Net framework 4.6!)

I agree. The test projects were blindly copied from the old api repo and are not updated and relevant anymore.

The clientApp project was my experiment to use the C# source generator to automatically generate simple api client from the swagger.json file. It is a console application that I used for testing the api locally during development of the api plugin. It is not necessary or useful for the api plugin users.
